### DISCUSSION TITLE
Issues with nivo.rocks website?

### DISCUSSION THREAD

**Discussion by ldalves** [plouc/nivo#2321] [General]
Hi!

It seems to me that this section of the site is not working properly, for example, it is not possible to type in the search input. 
Does this happen to you as well? 

https://nivo.rocks/components

I tested in different browser and it is the same.

Best regards, 

Leandro

**Comment by plouc** [comment]
Hi, thank you for reporting this issue, I'll have a look.

**Comment by SebasQuirogaUCP** [comment]
It seems to be functional

**Comment by plouc** [comment]
It has been fixed since.
